src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const tBody = document.querySelector('tbody');
+const arraySpecialists = [...tBody.children];
+const elementsTotal = document.querySelectorAll('th');
+
+for (let i = 0; i < elementsTotal.length; i++) {
+  elementsTotal[i].id = [i];
+}
+
+const SortTable = (e) => {
+  arraySpecialists.sort((a, b) => {
+    const itemA = a.cells[e.target.id].innerText;
+    const itemB = b.cells[e.target.id].innerText;
+
+    switch (e.target.innerText) {
+      case 'Name':
+        return itemA.localeCompare(itemB);
+      case 'Position':
+        return itemA.localeCompare(itemB);
+      case 'Age':
+        return itemA - itemB;
+      case 'Salary':
+        return itemA.slice(1).split(',').join('')
+          - itemB.slice(1).split(',').join('');
```

Comment:
use `default `case to handle the invalid sort type. And then let's think: in this default case we still iterate through the whole array. But should we? Let's optimize it. May be would be better to move `switch case` outside the `sort `